tests/ui/associated-types/associated-types-path-1.stderr

```diff
@@ -2,7 +2,9 @@ error[E0220]: associated type `A` not found for `T`
   --> $DIR/associated-types-path-1.rs:10:26
    |
 LL | pub fn f1<T>(a: T, x: T::A) {}
-   |                          ^ associated type `A` not found
+   |                          ^
+   |
+   = help: `T` has no associated type, try removing `A`
```

Comment:
Nor does this suggestion make sense.

Reply:
We already suggest what traits provide `A` below. We should just remove this help.

Reply:
Didn't thought much about it but yeah, if the user tries to use an associated type the compiler shouldn't tell them to remove it.